modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -454,7 +468,9 @@ public class ApiClient {
      * @return Api client
      */
     public ApiClient setConnectTimeout(int connectionTimeout) {
-        httpClient.setConnectTimeout(connectionTimeout, TimeUnit.MILLISECONDS);
+        httpClient = httpClient.newBuilder()
```

Comment:
One notable change in version 3.x is that a client cannot be modified after it has been created. The alternative is to create a _new_ client by converting back to a builder, modifying the relevant fields and the instantiating a new client. I've adopted this pattern throughout, where client mutations need to take place.